src/App.vue

```diff
@@ -84,6 +84,9 @@ export default {
     },
     tie() {
       return this.vueStars === this.reactStars
+    },
+    left() {
```

Comment:
Same as above... don't think we need this?

Reply:
same here)